src/styles/blocks/start-day.scss

```diff
@@ -0,0 +1,6 @@
+@each $day, $margin in $days {
+  .calendar--start-day--#{$day}
+  &:first-child {
+    margin-left: $margin * ($cell-size + $gap + 2 * $border-width);
```

Comment:
Make sure to deploy your page before re-requesting a review 
<img width="277" alt="Screenshot 2023-11-03 at 17 50 04" src="https://github.com/mate-academy/layout_calendar/assets/135232001/3bcda2e4-7551-4977-83b7-7d2b32549685">

Reply:
i don't know why i can't add my updates from VScode ;/